vm/rust/Cargo.toml

```diff
@@ -16,6 +16,7 @@ cached = "0.46.1"
 once_cell = "1.18.0"
 lazy_static = "1.4.0"
 
+# cairo-lang-* packages version should be in sync with starknet/rust/Cargo.toml cairo-lang-classes package
```

Comment:
Can we not remove the following packages? As they are not used in this rust code?

Reply:
Yep, removed it. It should speed up our compilation of Rust